AdyenComponents/Boleto/BoletoComponent.swift

```diff
@@ -196,6 +191,20 @@ public final class BoletoComponent: PaymentComponent, LoadingComponent, Presenta
     }
 }
 
+extension BoletoComponent: ViewControllerDelegate {
+
+    /// :nodoc:
+    public func viewDidLoad(viewController: UIViewController) {}
```

Comment:
we should call analytics here, right?

Reply:
Yeah, Boleto is missing it. Let me create a different PR for it